### ISSUE TITLE
API time string missing timezone information

### ISSUE THREAD

**Issue by pm17788** [wazuh/wazuh-api#305]
Hello,
When doing API calls (example: `/syscheck/:agentid/last_scan`), the timestamp returned is missing TZ data. 

Example output:
```json
{
  "error": 0,
  "data": {
    "start": "2019-02-08 13:44:47",
    "end": "2019-02-08 10:28:31"
  }
}
```

**Comment by mgmacias95** [comment]
Hello @pm17788,

Wazuh only uses UTC time, not only in syscheck/rootcheck information but in logs, alerts, etc. That's why the timezone information isn't shown.

Best regards,
Marta

**Comment by pm17788** [comment]
Hello @mgmacias95,

Thank you for the reply. Fair point about UTC. Might I ask for this to be explicitly stated on the API Reference pages (https://documentation.wazuh.com/current/user-manual/api/reference.html) - it wasn't obvious. 

A weaker argument is to explicitly encode it, because that removes ambiguity, but that's a bit more work :)

**Comment by crd1985** [comment]
Hello @pm17788 ,
we are planning to improve documentation to include a detailed description of the responses. We will consider your request.
Thanks a lot for your suggestion!

**Comment by mgmacias95** [comment]
Hello @pm17788,

We're doing a migration of our actual API to OpenAPI. This migration will return all dates in the [RFC3339](https://xml2rfc.tools.ietf.org/public/rfc/html/rfc3339.html#anchor14) format which includes timezone information.

This means this issue will be fixed at Wazuh 4.0 but it won't be fixed in 3.x releases.

Best regards,
Marta

**Comment by davidjiglesias** [comment]
Fixed in Wazuh v4.0.0
